### PR TITLE
MF-1455 - Update Versions of Protobuf

### LINKED ISSUE #1455: Bumping up versions of protobuf
**ENHANCEMENT**
1. Describe the enhancement you are requesting. Enhancements include:
   - tests
   - code refactor
   - documentation
   - research
   - tooling

Bumping up versions of `protobuf`. The current version of `protobuf` (e.g. in [CI script](https://github.com/mainflux/mainflux/blob/master/scripts/ci.sh)) should be updated. It needs an update.

2. Indicate the importance of this enhancement to you (must-have, should-have, nice-to-have).
nice-to-have

### PR DESCRIPTION
**Pull request by rodneyosodo**
Signed-off-by: rodneyosodo <socials@rodneyosodo.com>

### What does this do?
Updates versions of protobuf

### Which issue(s) does this PR fix/relate to?
Resolves #1455

### List any changes that modify/break current functionality
When updating the protobuf version, it is possible that some changes may break backward compatibility. The change in the protobuf version enforces that the message is passed in as a pointer since it contains a sync mutex. Therefore, any existing code that attempted to pass the message without a pointer would no longer function correctly.

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### Notes
N/A

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -78,7 +78,7 @@ test:
 
 proto:
 	protoc --gofast_out=plugins=grpc:. *.proto
-	protoc --gofast_out=plugins=grpc:. pkg/messaging/*.proto
+	protoc -I. --go_out=. --go_opt=paths=source_relative pkg/messaging/*.proto
```
Comment (dborovcanin) [inline Makefile:80]: The line above (gRPC part) is still using gogo Protobufs (gofast_out).

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Review by dborovcanin — DISMISSED** [review]

File: pkg/transformers/senml/transformer_test.go [hunk 2]
```diff
@@ -92,21 +92,24 @@ func TestTransformCBOR(t *testing.T) {
 	require.Nil(t, err, "Decoding CBOR expected to succeed")
 
 	tr := senml.New(senml.CBOR)
-	msg := messaging.Message{
+
+	// 82AD2169626173652D6E616D6522F956402369626173652D756E6974200A24F9490025F9564000646E616D650164756E697406F95CB0036331323307F958B002F9514005F94900AA2169626173652D6E616D6522F956402369626173652D756E6974200A24F9490025F9564000646E616D6506F95CB007F958B005F94900
```
Comment (drasko) [inline pkg/transformers/senml/transformer_test.go]: What is this? Some dead code? Or a regular comment?

**Review by drasko — CHANGES_REQUESTED** [review]

File: pkg/transformers/senml/transformer_test.go (hunk 2 above)
Comment (rodneyosodo) [reply pkg/transformers/senml/transformer_test.go]: It was initially there

**Review by drasko — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Comment by rodneyosodo** [comment]
Resolves https://github.com/mainflux/mainflux/issues/1287

**Pull request merged**